docs/app/src/examples.js

```diff
@@ -1,8 +1,9 @@
+'use strict';
+
 angular.module('examples', [])
 
 .directive('runnableExample', ['$templateCache', '$document', function($templateCache, $document) {
   var exampleClassNameSelector = '.runnable-example-file';
-  var doc = $document[0];
```

Comment:
Actually we can (and should) remove the `$document` dependency altogether.
The `$templateCache` too.